### PR TITLE
Removed `each()` function usage

### PR DESCRIPTION
**Pull request by YevSent**
Removed `each()` function usage
Updated static test to check `each()` usage

### Fixed Issues
1. magento-engcom/php-7.2-support#20: Usage of deprecated each() function

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: setup/src/Magento/Setup/Model/Installer.php [hunk 1]
```diff
@@ -337,7 +337,8 @@ public function install($request)
 
         $this->log->log('Starting Magento installation:');
 
-        while (list(, list($message, $method, $params)) = each($script)) {
+        foreach ($script as $item) {
+            list($message, $method, $params) = [$item[0], $item[1], $item[2]];
```
Comment (ihor-sviziev) [inline setup/src/Magento/Setup/Model/Installer.php]: I'd like to simplify it just with 
```php 
list($message, $method, $params) = $item;
```

File: dev/tests/static/testsuite/Magento/Test/Integrity/ComposerTest.php [hunk 2]
```diff
@@ -336,7 +336,8 @@ public function testComponentPathsInRoot()
             "If there are any component paths specified, then they must be reflected in 'replace' section"
         );
         $flat = $this->getFlatPathsInfo(self::$rootJson['extra']['component_paths']);
-        while (list(, list($component, $path)) = each($flat)) {
+        foreach ($flat as $item) {
+            list($component, $path) = [$item[0], $item[1]];
```
Comment (ihor-sviziev) [inline dev/tests/static/testsuite/Magento/Test/Integrity/ComposerTest.php]: I'd like to simplify it just with
```php
list($component, $path) = $item;
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: setup/src/Magento/Setup/Model/Installer.php (hunk 1 above)
Comment (YevSent) [reply setup/src/Magento/Setup/Model/Installer.php]: Done

File: dev/tests/static/testsuite/Magento/Test/Integrity/ComposerTest.php (hunk 2 above)
Comment (YevSent) [reply dev/tests/static/testsuite/Magento/Test/Integrity/ComposerTest.php]: Done

**Review by buskamuza — APPROVED** [review]

**Review by ihor-sviziev — APPROVED** [review]

**Pull request merged**